File: android-exercises/10 - options_menu/OptionsMenu_java/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_java/MainActivity2.java
package com.bugrahankaramollaoglu.optionsmenu_java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity2 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
    }

    public void goBack2(View view) {
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
    }
}

File: android-exercises/10 - options_menu/OptionsMenu_kotlin/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_kotlin/MainActivity2.kt
package com.bugrahankaramollaoglu.optionsmenu_kotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity2 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)
    }

    fun goBack2(view : View) {
        val intent : Intent = Intent(this@MainActivity2, MainActivity::class.java)
        startActivity(intent)
    }
}